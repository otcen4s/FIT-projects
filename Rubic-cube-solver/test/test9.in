465
316
316
654 332 141 322
324 332 146 551
321 555 646 551
264
264
214