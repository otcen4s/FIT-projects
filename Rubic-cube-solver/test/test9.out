465
316
316
654 332 141 322
324 332 146 551
321 555 646 551
264
264
214

466
311
311
655 225 441 322
326 335 446 551
326 335 446 551
264
264
211

666
311
311
255 225 443 211
226 335 443 255
226 335 444 355
664
664
111

322
311
311
255 226 334 111
226 336 444 155
226 336 444 155
664
664
555

333
112
112
226 334 111 255
226 336 444 155
226 336 444 155
664
664
555

112
112
112
226 333 144 555
226 333 144 555
226 333 144 555
664
664
664

111
111
111
222 333 444 555
222 333 444 555
222 333 444 555
666
666
666

